src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import { PeopleContext } from '../../Context';
+import classNames from 'classnames';
+import { useLocation } from 'react-router-dom';
+
+export const PeopleTable: React.FC = () => {
+  const { people } = useContext(PeopleContext);
+  const { pathname } = useLocation();
+  const sexColor = (sex: string) => {
+    return classNames({ 'has-text-danger': sex === 'f' });
+  };
```

Comment:
The `sexColor` function currently only handles the case for 'f'. Consider extending it to handle other cases like 'm' for better styling flexibility.